TestCases/parallel_regression_AD.py

```diff
@@ -168,6 +168,21 @@ def main():
     discadj_incomp_turb_NACA0012_sst.tol       = 0.00001
     test_list.append(discadj_incomp_turb_NACA0012_sst)
 
+    ####################################################################
+    ###  Disc. Adj. Axisymmetric RANS                                ###
+    ####################################################################
+
+    # Adjoint Axisymmetric RANS
+    discadj_axisymmetric_rans_nozzle           = TestCase('discadj_axisymmetric_rans')
+    discadj_axisymmetric_rans_nozzle.cfg_dir   = "axisymmetric_rans/air_nozzle"
+    discadj_axisymmetric_rans_nozzle.cfg_file  = "air_nozzle_adj.cfg"
+    discadj_axisymmetric_rans_nozzle.test_iter = 10
+    discadj_axisymmetric_rans_nozzle.test_vals = [10.000000, -10.391857, -15.524696, -7.715907, -17.350541]        
+    discadj_axisymmetric_rans_nozzle.su2_exec  = "parallel_computation.py -f"
```

Comment:
```suggestion
    discadj_axisymmetric_rans_nozzle.su2_exec  = "mpirun -n 2 SU2_CFD_AD"
    discadj_axisymmetric_rans_nozzle.no_restart = True
```

Reply:
I removed `MATH_PROBLEM= DISCRETE_ADJOINT` in my suggestion above, using SU2_CFD_AD directly ensures that a DA run is done. parallel_computation (to my knowledge) also executes SU2_SOL after SU2_CFD which is the major difference to this